package.json

```diff
@@ -39,28 +39,34 @@
     "@prisma/client": "5.9.1",
     "@types/bcrypt": "^5.0.2",
     "@types/bcryptjs": "^2.4.6",
-    "@types/express": "^4.17.21",
     "ajv": "^8.12.0",
     "bcryptjs": "^2.4.3",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "express": "^4.18.2",
+    "fluent-ffmpeg": "^2.1.2",
```

Comment:
Are you sure you need ffmpeg? This package requires binary ffmpeg executable as prerequisite, which means you need some extra effort like configuring in dockerfile. See if you can generate thumbnail, etc. without ffmpeg binary executable.

Reply:
I cannot find another package that works better for getting image video audio data......If you know one please let me know.